libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -87,6 +87,17 @@ void AP_Camera::take_picture()
     primary->take_picture();
 }
 
+// take a picture
+void AP_Camera::take_multiple_pictures(const float& time, const float& total_pictures, const float& seq)
```

Comment:
no need to pass these in as const references I think.  They're just simple floats so just pass-by-value.